index.html

```diff
@@ -9,6 +9,7 @@
   <link rel="stylesheet" type="text/css" href="css/mobile.css">
   <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.3/components/icon.min.css">
   <script   src="https://code.jquery.com/jquery-3.1.0.js"   integrity="sha256-slogkvB1K3VOkzAI8QITxV3VzpOnkeNVsKvtkYLMjfk="   crossorigin="anonymous" defer></script>
+  <script type="text/javascript" src="js/typed.js" defer></script>
```

Comment:
Let's put the defers right after script, but before the other attributes. Easier to distinguish.